site/en/docs/extensions/mv3/getstarted/index.md

```diff
@@ -129,7 +129,7 @@ green`"
 
 ## Introduce a user interface {: #user_interface }
 
-Extensions can have many forms of a [user interface][4]; this one will use a [popup][15]. Create and
+[User interfaces] for extensions can have many forms[4]; this one will use a [popup][15]. Create and
```

Comment:
In its current form this PR breaks the "user interfaces" link because it separates the link text (`[User interfaces]`)  from the link destination (`[4]`).

To fix this, you'll need to move the `[4]` over so it immediately follows the link text as in this suggestion:

```suggestion
[User interfaces][4] for extensions can have many forms; this one will use a [popup][15]. Create and
```